docs/insomnia/version-control-sync.md

```diff
@@ -6,73 +6,72 @@ category-url: get-started
 
-In this section, you will find documentation on [Sync](#sync) and [Snapshots](#snapshots). 
+In this section, you will find documentation on Collections [Sync](#sync) and [Snapshots](#snapshots).
 
 ## Sync
 
-{:.alert .alert-primary}
-**Note**: Sync is only available to Collections that exist inside a remote Project. 
-
-Sync provides the following abilities on top of the base Insomnia functionality:
+Sync for remote Collections provides the following abilities on top of the base Insomnia functionality:
 
 * Snapshot the contents of a request collection
 * Revert a request collection to a previous Snapshot
 * Share Snapshots across devices or with members of your team
 * Create and work on separate branches
 
-The main features of sync can be accessed from the dropdown at the top right of the application when you are in a Collection.
+The main features of sync (once enabled automatically or manually) can be accessed from the dropdown at the top right of the application when you are in a remote Collection.
 
 ![With a paid subscription, click on the large branch name button within a Collection.](/assets/images/version-control.png)
 _For versioning options on a Collection, click on the large branch name button._
 
-### Enable Sync
-
-Sync is only available to Collections that exist inside a [remote Project](/projects). When sync is enabled on a Collection inside a particular Project, all users that have access to the project also gain access to that Collection. 
+## Enable Sync
 
-You can manage the members in your Project from the [web dashboard](https://app.insomnia.rest/app/signup/).
+On creating a request Collection within a Remote Project, Insomnia will attempt to automatically enable sync and push an initial snapshot.
 
-#### Auto enable sync
+### Manually Enable Sync
 
-On creating a Collection within a remote Project, Insomnia will attempt to automatically enable sync and push an initial Snapshot.
-
-#### Manually enable sync
 If Insomnia is unable to or has not automatically enabled sync, then you can manually set this up. First click **Setup Sync** then **Create Local Project**. Then you can create and push your first Snapshot.
 
 {:.alert .alert-primary}
-**Note**: Can't see the Sync menu? Make sure you're logged in by clicking on the person icon in the top right corner of your application. If you see a logout option, you're logged in. 
+**Note**: Can't see the Sync menu? Make sure you're logged in by clicking on the person icon in the top right corner of your application. If you see a logout option, you're logged in.
+
+![Enable Git sync within a Collection by clicking the Setup Sync button.](/assets/images/setup-sync.png)
+_Enable Git sync within a Collection by clicking the Setup Sync button._
```

Comment:
We might want to update this screenshot. In the screenshot there is a "Share" button that shows, but that doesn't actually exist anymore. Screenshots will no doubt fall out of sync, but since this button was related to Version Control Sync as well it could be misleading.